xbmc/Application.cpp

```diff
@@ -823,8 +822,9 @@ bool CApplication::Initialize()
       else
       {
         // If no update is active disable all incompatible addons during start
+        incompatibleAddons.clear();
```

Comment:
This breaks a bit the work of `DisableIncompatibleAddons` as it is always empty :smile: 

Used when is not "AUTO_UPDATES_ON" before.